src/installation/guides/chroot.md

```diff
@@ -0,0 +1,303 @@
+as for the EFI System Partition).
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on the live images
+for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+For a UEFI booting system, make sure to create an EFI System Partition (ESP).
+The ESP should have the partition type "EFI System" (code `EF00`) and be
+formatted as FAT32 using [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+If you're unsure what partitions to create, create a 1GB partition of type `EFI
+System` (code `EF00`), then create a second partition of type `Linux Filesystem`
+(code `8300`) using the remainder of the drive.
+
+Format these partitions as FAT32 and ext4, respectively.
+
+```
+# mkfs.vfat /dev/sda1
+# mkfs.ext4 /dev/sda2
+```
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `dev/sda1` is the EFI
+System Partition:
+
+```
+# mount /dev/sda2 /mnt/
+# mkdir -p /mnt/boot/efi/
+# mount /dev/sda1 /mnt/boot/efi/
+```
+
+Initialize swap space, if desired, using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+Follow only one of the two following subsections.
+
+### The XBPS Method
+
+[Select an XBPS mirror](../../xbps/repositories/mirrors/index.md). For
+simplicity, save this URL to a shell variable for later use. Append `/current`
+to this repository URL to install a glibc distribution, or `/current/musl` for a
+musl distribution.
+
+Note: musl binaries are not available for i686.
+
+For example, this is the URL for glibc from a Tier 1 server:
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+XBPS also needs to know what architecture is being installed. Available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers. For
+example:
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Use [xbps-install](https://man.voidlinux.org/xbps-install.1) to bootstrap the
+installation by installing the `base-system` metapackage:
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r /mnt -R "$REPO" base-system
+```
+
+### The ROOTFS Method
+
+[Download a ROOTFS
+tarball](https://voidlinux.org/download/#download-installable-base-live-images-and-rootfs-tarballs)
+matching your architecture.
+
+Unpack the tarball into the newly configured filesystems:
+
+```
+# tar xvf void-[...]-ROOTFS.tar.xz -C /mnt
```

Comment:
After checking the Handbook for extant usage patterns, i've updated the style guide to mention the format for [placeholders](https://docs.voidlinux.org/contributing/void-docs/style-guide.html#placeholders) - could you please change this to `void-<...>-ROOTFS`?

Reply:
sounds good